mbuild/tests/test_lattice.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import pytest
+from mbuild.tests.base_test import BaseTest
+import mbuild as mb
+from lattice import Lattice
```

Comment:
This import will never work. You'll probably want to modify https://github.com/mosdef-hub/mbuild/blob/master/mbuild/__init__.py to add your new Lattice class

Also FYI, while `from foo import *` is generally frowned upon, you can explicitly control what gets imported via statements like this: https://github.com/mosdef-hub/mbuild/blob/master/mbuild/compound.py#L3